check-ntpoffset/lib/check-ntpoffset.go

```diff
@@ -206,3 +211,65 @@ func parseNTPOffsetFromChrony(out io.Reader) (offset float64, err error) {
 	}
 	return 0.0, fmt.Errorf("failed to get ntp offset")
 }
+
+func getNTPStratum() (string, error) {
+	ntpdName, err := detectNTPDname()
```

Comment:
`detectNTPDName` is also called in `getNTPOffset`.
Can this be called only once?

Reply:
I've changed  to get stratum and offset with one command only.